main.py

```diff
@@ -3,5 +1,46 @@
+import init_django_orm  # noqa: F401p
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = [("Western",), ("Action",), ("Dramma",)]
+    for genre_name, in genres:
+        Genre.objects.create(name=genre_name)
+        Genre.objects.filter(name="Dramma").update(name="Drama")
+
+    actors = [
+        ("George", "Klooney"),
+        ("Keanu", "Reeves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for first_name, last_name in actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
```

Comment:
This line is redundant because the update operation for changing 'Dramma' to 'Drama' is already performed inside the loop. Remove this line to avoid unnecessary database operations.